tools/get_metryc_footprint.py

```diff
@@ -0,0 +1,123 @@
+
+import sys
+import json
+import argparse
+from pathlib import Path
+import geopandas as gpd
+
+sys.path.append(str(Path(__file__).resolve().parent.parent))
+from reaskapi.metryc import Metryc
+
+
+def make_output_filename(parsed_args):
+
+    args = dict(parsed_args.__dict__)
+
+    terrain_map = {'full_terrain_gust': 'FT',
+                   'open_water': 'OW',
+                   'open_terrain': 'OT'}
+    args['terrain_correction'] = terrain_map[args['terrain_correction']]
+
+    if args['format'] == 'geotiff':
+        suffix = 'tiff'
+    else:
+        suffix = args['format']
```

Comment:
saves some lines if you move suffix and template above these blocks and set defaults there.